perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -53,6 +57,95 @@
                   'scratch_disk': True,
                   'num_machines': 1}
 
+GLOBALS_TEMPLATE = """
+[global]
+ioengine=libaio
+invalidate=1
+blocksize=4k
+direct=1
+runtime=10m
+time_based
+filename={filename}
+do_verify=0
+verify_fatal=0
+randrepeat=0
+size={size}
+
+"""
+
+SINGLE_JOB_TEMPLATE = """
+[{rwkind}-io-depth-{iodepth}]
+stonewall
+rw={rwkind}
+iodepth={iodepth}
+
+"""
```

Comment:
We've been using Jinja2 templates for dynamically-generated config files. Something like:

```
[global]
ioengine=libaio
invalidate=1
blocksize=4k
direct=1
runtime=10m
time_based
filename={{filename}}
do_verify=0
verify_fatal=0
randrepeat=0
size={{size}}

{% for rwkind in ('randread', 'randwrite') %}  # or an input variable
{% for iodepth in iodepths %}
[{{rwkind}}-io-depth-{{iodepth}}]
stonewall
rw={{rwkind}}
iodepth={{iodepth}}
{% endfor %}
{% endfor %}
```

I think that communicates the intent of the configuration slightly better, since the template text and composition aren't separated.